src/styl/imports/dark-theme.styl

```diff
@@ -0,0 +1,10 @@
+body.dark
+	background-color #222 !important
+
+	& #color-switch
+		color white !important
+
+	& h1, p, span
+		color white !important
+	& a
+		color #d11717 !important
```

Comment:
This file misses a ending new line, you need to run lint ! 

=> Lint should be ran automatically on commit with husky and lint-staged modules
Why is it not working for you ?

I think you should do a clean merge or rebase from upstream because it looks that you misses files. 
You may want to do a npm install again also.

Reply:
I  ran lint but it didn't show any errors! Guess it may be missing something. I will run npm again.

Reply:
Does it works for you now?